project.clj

```diff
@@ -83,6 +89,7 @@
 
   :profiles {:dev {:source-paths  ["dev"]
                    :dependencies  [[org.clojure/tools.namespace "0.2.4"]
+                                   [ch.qos.logback/logback-access "1.1.7"]
```

Comment:
Oh and it might be worth adding a comment above these two places where you've added logback-access, to mention that they should hopefully not be necessary in the master branch?